client/StatusTable/components/scss/DayRow.module.scss

```diff
@@ -0,0 +1,32 @@
+.status-table__day-row {
```

Comment:
try to keep one block in one file

Reply:
Ideally class `status-table__day-row` should be passed as a prop from its parent